www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,208 @@
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+/* On smaller screens, decrease text size */
+@media only screen and (max-width: 300px) {
+  .prev, .next,.text {font-size: 11px}
+}
+</style>
+</head></style>
+    </head>
+           <section>
+                <header id="top" class="header container">
+                    <div class="text-vertical-center">
+                        <h1 class="c2">
+                            Mel Bolen's Gallery
+                        </h1><a class="c3" href="#about"></a>
+                    </div>
+                </header>
+            </section>
+		<a href="index.html"><img src="./img/returntohomepage.jpg" class="center" /></a>
+	<!-- Slideshow container -->
+<div class="slideshow-container">
+
+  <!-- Full-width images with number and caption text -->
+  <div class="mySlides fade">
+    <div class="numbertext">1 / 5</div>
+    <img src="./img/bol2.jpg" style="width:100%">
+    <div class="text">Mel Bolen.</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">2 / 5</div>
+    <img src="./img/wife.jpg" style="width:100%">
+    <div class="text">Mel Bolen's wife.</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">3 / 5</div>
+    <img src="./img/mbolenmothersday.jpg" style="width:100%">
+    <div class="text">Mel Bolen mothersday.</div>
+  </div>
+  
+  <div class="mySlides fade">
+	<div class="numbertext">4 / 5</div>
+	<img src="./img/mbolenvase.jpg" style="width:100%">
+	<div class="text">Mel Bolen with a vase.</div>
+</div>
+
+   <div class="mySlides fade">
+	<div class="numbertext">5 / 5</div>
+	<img src="./img/mbolenandkaren.jpg" style="width:100%">
+	<div class="text">Mel Bolen and Karen Holden.</div>
+</div>
+  <!-- Next and previous buttons -->
+  <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+  <a class="next" onclick="plusSlides(1)">&#10095;</a>
+</div>
+<br>
+
+<!-- The dots/circles -->
+<div style="text-align:center">
+  <span class="dot" onclick="currentSlide(1)"></span> 
+  <span class="dot" onclick="currentSlide(2)"></span> 
+  <span class="dot" onclick="currentSlide(3)"></span> 
+  <span class="dot" onclick="currenctSlide(4)"></span>
+  <span class="dot" onclick="currentSlide(5)"></span>
+</div>
+<script>
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    var dots = document.getElementsByClassName("dot");
+    for (i = 0; i < slides.length; i++) {
+       slides[i].style.display = "none";  
+    }
+    slideIndex++;
+    if (slideIndex > slides.length) {slideIndex = 1}    
+    for (i = 0; i < dots.length; i++) {
+        dots[i].className = dots[i].className.replace(" active", "");
+    }
+    slides[slideIndex-1].style.display = "block";  
+    dots[slideIndex-1].className += " active";
+    setTimeout(showSlides, 9000); // Change image every 9 seconds
+}
+</script>
+
```

Comment:
There are end tags missing at the end of the file. These would likely come up if our tests were running, but they aren't because of that missing comma.